### PR TITLE
Removed preview mode from README.md

### PR DESCRIPTION
**Pull request by 123Haynes**
The preview mode has been removed from the code for quite a while.
This PR updates the `README.md` accordingly.

### CODE REVIEW

**Comment by jkroepke** [comment]
Instead that, take a note about https://github.com/mc1arke/sonarqube-community-branch-plugin

**Comment by 123Haynes** [comment]
@jkroepke oh I did. But the pull request decoration of the community branch plugin is not on par with this plugin yet.  
Also I don't really get what this PR has to do with the other plugin.

**Comment by jkroepke** [comment]
I saw 

> This plugin requires the BranchPlugin for SonarQube CE

but newly people would appreciate it, if you add a link

**Comment by 123Haynes** [comment]
@jkroepke ah! That's what you meant. No problem. I added a link 😃

File: README.md [hunk 1]
```diff
@@ -156,25 +156,7 @@ Works with `sonar-scanner` and `gradle`
...
-  only:
-    - /^feature\/*/
-  script:
-    - git checkout origin/master
-    - git merge $CI_COMMIT_SHA --no-commit --no-ff
-    - mvn --batch-mode verify sonar:sonar -Dsonar.host.url=$SONAR_URL -Dsonar.login=$SONAR_LOGIN -Dsonar.analysis.mode=preview -Dsonar.gitlab.project_id=$CI_PROJECT_PATH -Dsonar.gitlab.commit_sha=$CI_COMMIT_SHA -Dsonar.gitlab.ref_name=$CI_COMMIT_REF_NAME
-```
-
-With quality gate
+This plugin requires the [Community Branch Plugin](https://github.com/mc1arke/sonarqube-community-branch-plugin)  for SonarQube CE
```
Comment (lucasoares) [inline README.md:159]: Based on [sonarqube-community-branch-plugin](https://github.com/mc1arke/sonarqube-community-branch-plugin) we should have to configure the `sonar.branch.name` parameter to enable PR decoration. We should add those examples?

Comment (lucasoares) [reply README.md:159]: I see..

I don't know if this is necessary.. Thanks!

**Pull request merged**